docs/release-notes.rst

```diff
@@ -6,6 +6,80 @@
  Release Notes
 ###############
 
+**************
+ Version 0.21
+**************
+
+Version 0.21.0
+==============
+
+**Release Date:** March 27, 2023
+
+**Breaking Changes**
+
+-  Cluster: K80 GPUs are no longer supported.
+
+-  Remove all old PATCH endpoints under `/agents*`. This covers the PATCH APIs for enabling and
+   disabling slots. Users should use the new APIs under `/api/v1/agents`
+
+-  Trial API: ``on_validation_step_start`` and ``on_validation_step_end`` callbacks on
+   ``PyTorchTrial`` and ``DeepspeedTrial`` were deprecated in 0.12.12 (Jul 2020) and have been
+   removed. Please use ``on_validation_start`` and ``on_validation_end`` instead.
+
+-  ``records_per_epoch`` has been dropped from PyTorch code paths. We were previously using this
+   value internally to estimate epoch lengths. We are now using the chief worker's epoch length as
+   the epoch length.
+
+-  ``average_training_metrics`` is no longer configurable. This value previously defaulted to false
+   and was dropped to simplify the training API. We always average training metrics now.
+
+-  Trial API: the unused ``latest_training`` field has been removed from ``GetTrial`` and
+   ``GetExperimentTrials`` APIs due to slow performance.
+
+**Bug Fixes**
+
+-  CLI: Fix an issue where ``det user change-password`` will return an authentication error when
+   trying to change current user's password.
+
+**Improvements**
+
+-  CLI: Default to provisioning Nvidia T4 GPU instances.
+
+   -  Command line deployments will now default to provisioning Nvidia T4 GPU instances instead of
```

Comment:
```suggestion
-  CLI: Command-line deployments will now default to provisioning Nvidia T4 GPU instances instead of
```
and adjust indentation of the following lines.